WorkLog.txt

```diff
@@ -0,0 +1,58 @@
+WorkLog.txt
+
+01. v0.0.1 Upgrade to Ruby 3.0.2 and Rails 7.0.3  17June2023
+   01. Setup -
+    Change .ruby-version and Gemfile  for new Ruby & Rails versions.
+    Remove Gemfile.lock
+    Change Gemfile faker entry to use main in place of master
+    bundle install
+      -- looks successful --
+
+   02. Database setup - 
+     Ran   bundle e rails db:setup  --> failed with:
+      error Couldn't find an integrity file
+      error Found 1 errors.
+
+      ========================================
+        Your Yarn packages are out of date!
+        Please run `yarn install --check-files` to update.
+      ========================================
+
+      ...after 'yarn install ....'  retried db:setup --- worked OK.
+
+   03. Start server - bundle e rails s
+       From browser, navigate to localhost:3000
+       ...after long initialization stage (2 min), the PrecisePlay spash screen is displyed.
+
+... lost some of this WorkLog ??  .....
+
+   04. Attempting from Postman:  localhost:3000/api/v1/sign_up
+       Fails with: Can't verify CSRF token authenticity.
+
+       Added to registrations_controller.rb
+         skip_before_action :verify_authenticity_token, :only => :create
+       ?? now CRSF issue is gone ???
+
+   05. The "sign_up" fails.
+       Added logger.info .... #{user.errors.inspect} to else branch after if user.save
+       ...fournd that a) needs email, b) needs at least 6 char pwd.
+       After some adjustments to Postman payload and registrations_controller.rb,
+       POST to sigh_up works.
+
+   06. Test with Postman:  POST localhost:3000/api/v1/sign_up
+       Body - set as JSON
+       {
+        "user": {
+                "user_id": "js",
+                "password": "js1111",
+                "password_confirmation": "js1111",
+                "nick_name": "Johnny",
+                "last_name": "Smith",
+                "first_name": "John",
+                "email": "js0@gmail.com",
+                "cell_number": "123-456-7890",
+                "admin_flg": "FALSE",
+                "pin": "1111"
+                }
+        }
+
```

Comment:
Depending on what route we go with the work log, we may remove this file. But we should definitely remove the three `.DS_Store` files below.

Reply:
I've put the .DS_Store files into .gitignore - I'll work on getting them out of the current repo.

Reply:
There were .DS_Store files in several directories.  These have been removed from the repo.